web/dissolution/i18n/locales/en-CA.ts

```diff
@@ -1,8 +1,44 @@
+/* eslint-disable max-len */
 export default {
+  label: {
+    addToLedger: 'Add to ledger',
+    addToLedgerOpt: 'Add to ledger (Optional)',
+    chooseAnEndDate: 'Choose an end date',
+    delayDate: 'Delay the Time of Dissolution or Cancellation',
+    delayFor: 'Delay For',
+    important: 'Important',
+    selectADate: 'Select a date',
+    sixMonths: '6 Months'
+  },
   page: {
```

Comment:
You will need to follow the same structure as person roles inside here (filingType=dissolution, filingSubType=delay)
```
  [filingType]: {
      [filingSubType]: {...
  ```